episodes/13-synchronization.md

```diff
@@ -2,22 +2,21 @@
 title: "Synchronising tasks"
 teaching: 60
 exercises: 30
-questions:
+---
+
+:::::::::::::::::::::::::::::::::::::: questions
 - "How should I access my data in parallel?"
-objectives:
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
 - "First objective."
-keypoints:
-- "You can explicitly synchronise tasks with `sync` statement."
-- "You can also use sync and atomic variables to synchronise tasks."
----
+::::::::::::::::::::::::::::::::::::::::::::::::
 
-The keyword `sync` provides all sorts of mechanisms to synchronise tasks in
-Chapel.
+The keyword `sync` provides all sorts of mechanisms to synchronise tasks in Chapel.
```

Comment:
`sync` is a **_type qualifier_**, not a "keyword."

Reply:
Actually, it is either a statement or a type qualifier, depending on its use. This chapter covers both. I inserted the introductory sentence to clarify that. If you are not happy with *keyword*, please suggest another term.